src/SortableContainer/index.js

```diff
@@ -584,8 +584,8 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
       for (let i = 0, len = nodes.length; i < len; i++) {
         const {node} = nodes[i];
         const index = node.sortableInfo.index;
-        const width = node.offsetWidth;
-        const height = node.offsetHeight;
+        const width = node.getBoundingClientRect().width;
+        const height = node.getBoundingClientRect().height;
```

Comment:
I don't think `getBoundingClientRect` is "free". Thoughts on writing this
```
const {width, height} = node.getBoundingClientRect()
```
?

Could be written similarly above.